shared/android/app/src/debug/res/values/strings.xml

```diff
@@ -1,3 +1,3 @@
 <resources>
-    <string name="app_name">K̴̩̾è̴ͅy̶̢̑b̴̯̔a̶͚͑s̶̱͠e̴͍̅ DEBUG</string>
+    <string name="app_name">KeybaseDEBUG</string>
```

Comment:
This actually breaks some android toolchain stuff which makes using the emulator fail